scripts/storage-emulator-integration/tests.ts

```diff
@@ -263,9 +263,51 @@ describe("Storage emulator", () => {
           });
         });
 
-        // TODO(abehaskins): This test is temporarily disabled due to a credentials issue
-        it.skip("should handle large (resumable) uploads", async () => {
-          await testBucket.upload(largeFilePath, { resumable: true });
+        it.only("should handle large (resumable) uploads", async () => {
```

Comment:
Delete `only`

Reply:
By the way, what does this unit test have to do with large uploads?

Reply:
Good call, deleted only

Reply:
Nothing in particular with large uploads, this is more to test for resumable functionality. I'll remove the reference to a large file to simplify this.